org.kde.filelight.json

```diff
@@ -6,10 +6,11 @@
     "command": "filelight",
     "finish-args": [
         "--device=dri",
-        "--filesystem=host",
         "--share=ipc",
         "--socket=fallback-x11",
-        "--socket=wayland"
+        "--socket=wayland",
+        "--nofilesystem=host",
```

Comment:
I don't think this is needed if we only have the line below

Reply:
You're right!